Pod/Classes/APMAlertController.swift

```diff
@@ -28,7 +30,7 @@ import SnapKit
     fileprivate var topScrollViewHeightConstraint: Constraint?
     fileprivate let contentView = UIView()
     fileprivate var anyTitleObject: AnyObject
-    fileprivate var topTitleMessageSeparatorConstraint: Constraint?
+    fileprivate var titleMessageSeparatorConstraint: Constraint?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/38fcdf550b8399a39a87fffc4ef0d6d6e45b99b7/Pod/Classes/APMAlertController.swift#L33"></span>Variable name should be between 2 and 24 characters long: ‘titleMessageSeparatorConstraint’</td>
    </tr>
  </tbody>
</table>